Dockerfile

```diff
@@ -14,5 +14,9 @@ COPY --from=build-env --chown=nonroot:nonroot /go/bin/app /
 # Run as a non root user.
 USER nonroot
 
+ARG BUSYBOX_VERSION=1.31.0-i686-uclibc
```

Comment:
"i686" I guess it is the architecture for the binary? Then I'm afraid it will limit this container to x86 only, whereas this container is also available on ARM device.